src/services/todoService.ts

```diff
@@ -0,0 +1,15 @@
+import { TodoItemType } from "../types/todoTypes";
+
+const getTodoList = () => {
+	const data = localStorage.getItem("todo");
+	return data ? JSON.parse(data) : [];
+}
+
+const setTodoList = (newTodoList: TodoItemType[]) => {
+	localStorage.setItem("todo", JSON.stringify(newTodoList));
+}
```

Comment:
Although this is fine... But this is rightly bound to this app.

Consider making it more generic. this service should've methods like: 
- addtodo
- gettodos
- updatetodo
- deletetodo

And may be a db service underneath that takes care of dealing with storage.
Benefits of this is that in future if you change your db... Your UI code remains same.